### PR TITLE
Feat/manager-9330: containers actions list

### PR DESCRIPTION
**Pull request by mohammed-zahaf**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/MANAGER-9069`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-9330
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description
1. add actions list
2. manage availability of actions list
3. implement add user modal
4. implement remove container objects
5. complete remove container
6. complete archive modal feature
7. complete restore modal feature
8. add guides for add an archive

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archive.routing.js [hunk 1]
```diff
@@ -120,9 +90,9 @@ export default /* @ngInject */ ($stateProvider) => {
         }),
 
       trackPage: /* @ngInject */ (atInternet) => (hit) =>
-        atInternet.trackPage({
+        atInternet.trackClick({
           name: `${COLD_ARCHIVE_TRACKING_PREFIX}::${hit}`,
-          type: 'action',
+          type: 'page',
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archive.routing.js:95]: **nitpick:** `trackClick` and `type:page` does not make a good combination.
Please refer https://www.npmjs.com/package/ng-at-internet/v/3.1.1#click-tracking

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archive.routing.js (hunk 1 above)
Comment (mohammed-zahaf) [reply packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archive.routing.js:95]: I guess the current implementation is the right one:
if we refer to ng-at-internet module w have those possibilities:
`
atInternet.trackClick({
    name: "my-click", // name of action
    type: "action",   // event type (optional) Possible values : action/navigation/download/exit
    level2: "1",      // (optional) if not configured in defaults you must specify your project id
    visitorId: "123"  // (optional) set identifiedVisitor.id sent with each hit
});
`

**Review by stif59100 — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/users/download-rclone/download-rclone.controller.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+            },
+          }),
+        );
+      })
+      .catch((err) =>
+        this.goBack(
+          this.$translate.instant(
+            'pci_projects_project_storages_cold_archive_users_download-rclone_error_rclone',
+            {
+              message: get(err, 'data.message', null),
```
Comment (ghyenne) [inline packages/manager/modules/pci/src/projects/project/storages/cold-archive/users/download-rclone/download-rclone.controller.js:73]: Same of https://github.com/ovh/manager/pull/8440#discussion_r1036792492

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/users/download-rclone/download-rclone.routing.js [hunk 3]
```diff
@@ -3,62 +3,55 @@ import {
...
+        userId,
+      ) => PciStoragesColdArchiveService.get(projectId, userId),
+      regions: /* @ngInject */ () => {
+        return [COLD_ARCHIVE_DEFAULT_REGION];
+      },
+      storageS3Regions: /* @ngInject */ () => {
+        return [COLD_ARCHIVE_DEFAULT_REGION];
+      },
+      rcloneGuide: /* @ngInject */ (coreConfig) => {
+        return get(RCLONE_GUIDE, coreConfig.getUser().ovhSubsidiary);
```
Comment (ghyenne) [inline packages/manager/modules/pci/src/projects/project/storages/cold-archive/users/download-rclone/download-rclone.routing.js:40]:
```suggestion
        return RCLONE_GUIDE[coreConfig.getUser().ovhSubsidiary] || RCLONE_GUIDE.DEFAULT;
```
You can add the default value if there is a missing subsidiary.

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/containers.controller.js [hunk 4]
```diff
@@ -58,4 +61,71 @@ export default class PciStoragesContainersUsersController {
       type: 'navigation',
     });
   }
+
+  static isActionsAvailable(container) {
+    const Ctrl = PciStoragesContainersUsersController;
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/containers.controller.js:66]: **nitpick:** Can we rename the controller class name to `PciStoragesContainersController`? the file indicates just containers and not related to users.

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/add-user/translations/Messages_fr_FR.json [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  "pci_projects_project_storages_coldArchive_containers_addUser_description_container_step_1": "Définissez le rôle de votre utilisateur sur le conteneur {{value}}.",
+  "pci_projects_project_storages_coldArchive_containers_addUser_description_object_step_1": "Définissez le rôle de votre utilisateur sur l'objet {{value}}.",
+  "pci_projects_project_storages_coldArchive_containers_addUser_submit_label": "Confirmer",
+  "pci_projects_project_storages_coldArchive_containers_addUser_next_label": "Suivant",
+  "pci_projects_project_storages_coldArchive_containers_addUser_cancel_label": "Annuler",
+  "pci_projects_project_storages_coldArchive_containers_addUser_back_label": "Retour",
+  "pci_projects_project_storages_coldArchive_containers_addUser_right_admin": "Administrator",
+  "pci_projects_project_storages_coldArchive_containers_addUser_right_readWrite": "Read and write",
+  "pci_projects_project_storages_coldArchive_containers_addUser_right_readOnly": "Read only",
+  "pci_projects_project_storages_coldArchive_containers_addUser_right_deny": "Deny",
```
[the remark this answers is not in the document]
Comment (mohammed-zahaf) [reply packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/add-user/translations/Messages_fr_FR.json:16]: Yep it was same for Object Storage project.

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/delete-objects/delete-objects.controller.js [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+      })
+      .catch((err) => {
+        this.trackPage('containers::container::delete::confirm_error');
+
+        return this.goBack(
+          this.$translate.instant(
+            'pci_projects_project_storages_cold_archive_containers_container_delete_objects_error_delete',
+            {
+              containerName: this.container?.name,
+              message: get(err, 'data.message', null),
```
[the remark this answers is not in the document]
Comment (mohammed-zahaf) [reply packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/delete-objects/delete-objects.controller.js:68]: complete

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/users/download-rclone/download-rclone.controller.js (hunk 2 above)
Comment (mohammed-zahaf) [reply packages/manager/modules/pci/src/projects/project/storages/cold-archive/users/download-rclone/download-rclone.controller.js:73]: complete

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/users/download-rclone/download-rclone.routing.js (hunk 3 above)
Comment (mohammed-zahaf) [reply packages/manager/modules/pci/src/projects/project/storages/cold-archive/users/download-rclone/download-rclone.routing.js:40]: completed

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/containers.controller.js (hunk 4 above)
Comment (mohammed-zahaf) [reply packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/containers.controller.js:66]: completed

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/delete/delete.controller.js [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+          ),
+        );
+      })
+      .catch((err) => {
+        this.trackPage('containers::container::delete::confirm_error');
+        return this.goBack(
+          this.$translate.instant(
+            'pci_projects_project_storages_containers_container_delete_error_delete',
+            {
+              message: get(err, 'data.message', null),
```
[the remark this answers is not in the document]
Comment (mohammed-zahaf) [reply packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/delete/delete.controller.js]: Completed

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archive.routing.js (hunk 1 above)
Comment (mohammed-zahaf) [reply packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archive.routing.js]: Completed

**Review by ghyenne — APPROVED** [review]

**Pull request merged**